# tak_server_ip: "192.168.50.86"
# tak_server_port: 8087
cot_publisher_node:
  ros__parameters:
    tak_server_ip: "192.168.50.86"
    tak_server_port: 8087
    publish_rate_hz: 1.0
    retry_timeout_sec: 5.0
    max_retries: 10
    robot_uid: "cot_publisher_node"